resources/access.rb

```diff
@@ -24,18 +24,18 @@
 property :access_user,   String, required: true, default: 'postgres'
 property :access_addr,   [String, nil], default: nil
 property :access_method, String, required: true, default: 'ident'
-property :notification,  Symbol, required: true, default: :reload
```

Comment:
Why don't let the user the possibilty to choose trigger method for the service ? May be, I don't want to restart postgresql service (in cluster env) when I add/remove an user.

Reply:
If I understand just call notification in postgresql_access resource call like we do with chef resources ?
I agree but I have a brain conflict by thinking that I will need to notify each time I call the resource 🤣. If you have i.e 50 entries in your hba file, there will be 50 notifications too.

Reply:
There won't be in terms of service notifications. Chef compacts them at the end of the run unless they are flagged as `:immediate` because by default all notifications are `:delayed` in resources.